### DISCUSSION TITLE
Problema na projeção de mídias

### DISCUSSION THREAD

**Discussion by IcaroCircundes** [sircharlo/meeting-media-manager#2561] [General]
Prezados, ao projetar as mídias elas ficam com um zoom bem ampliado. Mesmo após alterar as configurações da segunda tela para resolução, a cada reunião ele volta a dar um zoom nas mídias. E ele não altera o idioma para português Brasil no layout da reunião, e ainda está incluindo todas as imagens mesmo após ativarmos a opção de excluir essas mídias.
![Captura de tela 2024-08-13 215358](https://github.com/user-attachments/assets/5a0226e1-bf29-4d23-a0d2-38eeb2170c14)
![Captura de tela 2024-08-13 215340](https://github.com/user-attachments/assets/77e4b570-c133-4805-ad95-fbccd21c6af9)
![Captura de tela 2024-08-13 215304](https://github.com/user-attachments/assets/9d6ff0b6-9fbd-4127-a852-ce12ce5d124b)

**Comment by e-Feitosa** [comment]
Prezado, o problema sobre a tradução, será corrigido na versão nova (aguarde), e quando ao outro problema, é uma questão de configuração entre a sua máquina (PC) e a tv/telão. Você está usando o OBS Studio? Se sim, nas configurações de saída de vídeo, você tem a opção de alterar para a mesma resolução que está recebendo a imagem no palco.

**Reply by IcaroCircundes** [reply]
Muito obrigado pelo feedback, no caso do zoom no telão, não estamos usando o obs. O que te os feito é todos os dias antes da reunião a gente precisa alterar a resolução da tela pq ele fica muito grande e corta as imagens exibidas. E é somente com o m3 no JW funciona normal. Eu tentei reinstalar pra ver se poderia resolver, porém quando reinstala ele volta com a mesma configuração que deixamos.

**Reply by e-Feitosa** [reply]
Entendi, qualquer coisa, me envia uma mensagem inbox e combinamos para revisar as configurações que está usando.

**Comment by IcaroCircundes** [comment]
Muito obrigado, vô precisar, o sistema é excelente pra reuniões

Em qui., 15 de ago. de 2024 09:05, Feitosa ***@***.***>
escreveu:

**Reply by e-Feitosa** [reply]
@IcaroCircundes, não consegui falar diretamente com você para combinarmos, me envia uma mensagem no e-mail suporteteocratico arroba gmail ponto com e aí combinamos um dia e horário para conectar aí e tentar ajustar
